snippets/latex/tikz.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
For these to be accessible in the snippet engine, please add a new entry to the `package.json`. The language can be something like `"language": "latex-tikz"` to avoid overloading the default LaTeX snippets. 

If you have questions, please ping.